### PR TITLE
Adding cache buster to index.html #445

### PR DESCRIPTION
**Pull request by shech2**
## Summary by CodeRabbit

- **New Features**
  - Implemented cache busting for improved content freshness.

- **Refactor**
  - Updated script references for better cache management.

- **Tests**
  - Renamed login tests to subscribe tests to better reflect their purpose.

- **Documentation**
  - Added meta tags for cache control in the HTML document.

### CODE REVIEW

**Comment by shech2** [comment]
with a timestamp for today's date , need to be changed every month for now , im thinking of a automatic solution instead of this static one.

File: index.html [hunk 1]
```diff
@@ -273,5 +273,6 @@ <h1><span class="name"></span> Error</h1>
                   arrow_2_circlepath</i></button>
     </nav>
 </template>
+<script type="text/javascript" src="/cacheBuster.ts"></script>
```
Comment (daonb) [inline index.html:276]: I'm afraid this won't work. Any javascript will run AFTER the browser already cached index.html and therefore will not validate the cache.
I believe the only solution is to use the cache meta tags.

Comment (shech2) [reply index.html:276]: i Tired and the app wont load for some reason so i tried to bypass it , i will try again .

File: index.html [hunk 2]
```diff
@@ -13,7 +16,21 @@
     <meta name="apple-touch-fullscreen" content="yes" />
     <meta name="apple-mobile-web-app-title" content="Terminal7" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
+
     <script type="module" src="/main.ts"></script>
+
+    <script>
```
Comment (daonb) [inline index.html:22]: I don't believe we need this script anymore.

Comment (shech2) [reply index.html:20]: Done

Comment (daonb) [reply index.html:20]: It's still here

File: index.html [hunk 3]
```diff
@@ -13,7 +16,19 @@
     <meta name="apple-touch-fullscreen" content="yes" />
     <meta name="apple-mobile-web-app-title" content="Terminal7" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
-    <script type="module" src="/main.ts"></script>
```
Comment (shech2) [inline index.html:16]: Deleted

Comment (shech2) [reply index.html:16]: @daonb

**Comment by shech2** [comment]
i think this PR can be merged , we do not need tests for this i think.

**Pull request merged**